libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -295,6 +302,11 @@ void ContractCompiler::appendFunctionSelector(ContractDefinition const& _contrac
 		appendCalldataUnpacker(functionType->parameterTypes());
 		m_context.appendJumpTo(m_context.functionEntryLabel(functionType->declaration()));
 		m_context << returnTag;
+		m_context.adjustStackOffset(
+			CompilerUtils(m_context).sizeOnStack(functionType->returnParameterTypes()) -
+			CompilerUtils(m_context).sizeOnStack(functionType->parameterTypes()) -
+			1
```

Comment:
Why is against?

```
m_context.adjustStackOffset(
			-1
			-(CompilerUtils(m_context).sizeOnStack(functionType->returnParameterTypes()))
			+(CompilerUtils(m_context).sizeOnStack(functionType->parameterTypes()))
);
```

On one side it looks uglier, but shows the intent better.